test-js/unit/prompting/PromptPanelSpec.js

```diff
@@ -546,14 +546,20 @@ define([ 'dojo/number', 'dojo/i18n', 'common-ui/prompting/PromptPanel',
             }
           });
           comp.type = "ScrollingPromptPanelLayoutComponent";
-          spyOn(window, "$");
+
+          // TODO: Find alternative for:
+          //spyOn(window, "$");
```

Comment:
I don't think that rewrite will work: you can intercept the holder property, placing a spy function instead of jquery, but that doesn't say a thing about the actual jquery function being called...